client/vscode/src/inlineCompletionActivation.ts

```diff
@@ -45,6 +45,8 @@ export const CodewhispererInlineCompletionLanguages = [
 
 export function registerInlineCompletion(languageClient: LanguageClient) {
     const inlineCompletionProvider = new CodeWhispererInlineCompletionItemProvider(languageClient)
+    // POC-NEP: The displayName parameter is not supported in this version of VSCode API
+    // We'll need to add it when the API is updated
```

Comment:
This comment should explain the purpose of the displayName property and what it should be set to.